dissect/target/plugins/os/windows/registry.py

```diff
@@ -204,7 +204,7 @@ def root(self) -> KeyCollection:
 
     @internal
     @lru_cache(4096)
-    def key(self, key: Optional[str] = None) -> KeyCollection:
+    def key(self, key: str = "") -> KeyCollection:
```

Comment:
Why not use `key: Optional[str] = None` and use `key = key or ""` as the first line of the method?